### PR TITLE
Feat/backend

### PR DESCRIPTION
**Pull request by domdepasquale**

### CODE REVIEW

File: helm-charts/falcon-sensor/values.yaml [hunk 1]
```diff
@@ -196,6 +196,7 @@ falcon:
   app:
   trace: none
   feature:
+  backend:
```
Comment (redhatrises) [inline helm-charts/falcon-sensor/values.yaml:199]: Let's go ahead and set `kernel` here.

File: helm-charts/falcon-sensor/Chart.yaml [hunk 2]
```diff
@@ -15,12 +15,12 @@ type: application
...
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.17.18
+version: 1.17.19
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: 1.17.18
+appVersion: 1.17.19
```
Comment (redhatrises) [inline helm-charts/falcon-sensor/Chart.yaml:23]: Since this requires a specific sensor, I wonder if we should bump to `1.18`?

Comment (domdepasquale) [reply helm-charts/falcon-sensor/Chart.yaml]: Incoming change

File: helm-charts/falcon-sensor/values.yaml (hunk 1 above)
Comment (domdepasquale) [reply helm-charts/falcon-sensor/values.yaml]: Incoming change

**Comment by domdepasquale** [comment]
Setting to DRAFT as I don't want it merged quite yet

**Comment by redhatrises** [comment]
Ack

**Pull request merged**